### PR TITLE
CI: Add Windows Arm64 target

### PR DESCRIPTION
**Pull request by EwoutH**
Adds the `aarch64-pc-windows-msvc` target to the build matrix of the Windows job.

### CODE REVIEW

**Comment by kjvalencik** [comment]
@EwoutH Thanks for the contribution!

Did you encounter issues building for Windows Arm64? Our build matrix is already kind of large and if we haven't experienced a platform specific regression, we might want to hold off adding this.

**Comment by dennisameling** [comment]
Just wanted to chime in here to mention that Neon is working correctly on Windows arm64 with the N-API backend! 🚀  https://github.com/signalapp/ringrtc/pull/12

**Comment by EwoutH** [comment]
Rebased and tuned the Rust install a bit. All Arm64 (Aarch64) builds are still failing, I will investigate a bit more later this week.

File: .github/workflows/windows.yml [hunk 1]
```diff
@@ -15,23 +15,27 @@ jobs:
     runs-on: windows-latest
 
     strategy:
+      fail-fast: false
       matrix:
-        node-version: [10.x, 12.x, 14.x, 15.x]
```
Comment (kjvalencik) [inline .github/workflows/windows.yml:19]: FYI, Neon follows LTS and will support Node 10 until it goes out of maintenance 4/30.

I support removing it from the build matrix, but we may want to defer until after or closer to the date.

Comment (EwoutH) [reply .github/workflows/windows.yml:19]: Good point. Maybe we could go from a full factorial matrix to a more sparse configuration. Not every node version would have to be tested on every Rust toolchain and every target.

Comment (kjvalencik) [reply .github/workflows/windows.yml:19]: I would like to do that for the ARM builds. We can do a smaller matrix on PRs and the full matrix on merge to main.

**Pull request closed without merging**